scripts/utils/migrate/fetch-redirects.js

```diff
@@ -0,0 +1,35 @@
+const fetch = require('node-fetch');
+const logger = require('../logger');
+const { paginate } = require('./docs-api');
+
+require('dotenv').config();
+
+const fetchRedirects = async (nodeId) => {
+  const url = `https://docs-dev.newrelic.com/api/migration/redirects?id=${encodeURIComponent(
+    `node/${nodeId}`
+  )}`;
+
+  try {
+    const res = await fetch(url, {
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Phpshield-Key-Disable': process.env.ACQUIA_DEV_PHP_SHIELD_KEY,
+      },
+    });
+    const json = await res.json();
+    const { redirects } = json;
+    return redirects;
+  } catch (e) {
+    logger.error(`Error, could not fetch ${url}: ${e}`);
+  }
+};
+
+const fetchAllRedirects = async () => {
+  const redirects = await paginate('/api/migration/redirects', {
+    perPage: 300,
+  });
+
+  return redirects;
+};
```

Comment:
[nit] Since `paginate` returns a promise, you can simplify this:

```suggestion
const fetchAllRedirects = () => {
  return paginate('/api/migration/redirects', {
    perPage: 300,
  });
};
```

Reply:
refactored that file a bit so have to now i'm awaiting the result of `paginate` to run the grouping function on it